src/jumpstart/JumpstartIntroScreen.tsx

```diff
@@ -0,0 +1,145 @@
+import { jumpstartIntroSeen } from 'src/jumpstart/slice'
+import { useAddAssetsActions } from 'src/jumpstart/useAddAssetsActions'
+import { replace } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { jumpstartSendTokensSelector } from 'src/tokens/selectors'
+
+type Props = NativeStackScreenProps<StackParamList, Screens.JumpstartIntroScreen>
+
+/**
+ * The component is set up to follow specific conditions:
+ *  - if user has no assets – ALWAYS SHOW intro
+ *  - if user has any assets but intro screen hasn't been seen – SHOW intro
+ *  - if user has any assets and has already seen intro screen –
+ *    this screen should be skipped in favor of JumpstartEnterAmount
+ */
+export default function JumpstartIntroScreen({ navigation }: Props) {
+  const { t } = useTranslation()
+  const dispatch = useDispatch()
+  const insets = useSafeAreaInsets()
+  const addAssetsBottomSheetRef = React.useRef<BottomSheetRefType>(null)
+
+  const addAssetsActions = useAddAssetsActions()
+  const introSeen = useSelector(jumpstartIntroHasBeenSeenSelector)
+  const tokens = useSelector(jumpstartSendTokensSelector)
+  const noTokens = !tokens.length
+
+  function onButtonClick() {
+    if (noTokens) {
+      AppAnalytics.track(JumpstartEvents.jumpstart_add_assets_show_actions)
+      addAssetsBottomSheetRef.current?.snapToIndex(0)
+      return
+    }
+
+    replace(Screens.JumpstartEnterAmount)
+  }
+
+  // Track back button click
+  React.useEffect(() => {
+    const unsubscribe = navigation.addListener('beforeRemove', () => {
+      AppAnalytics.track(JumpstartEvents.jumpstart_intro_back)
+    })
+
+    return unsubscribe
+  }, [navigation])
+
+  // Whenever user sees intro screen for the first time – store the flag in redux and track it in analytics
+  React.useEffect(() => {
+    if (!introSeen) {
+      AppAnalytics.track(JumpstartEvents.jumpstart_intro_seen)
+      dispatch(jumpstartIntroSeen())
+    }
+  }, [introSeen, dispatch])
+
+  return (
+    <SafeAreaView style={styles.safeAreaContainer} edges={['top']}>
+      <PalmSharp style={styles.palmImage} />
+      <WaveCurve style={styles.waveImage} />
+      <CustomHeader style={styles.header} left={<BackButton />} />
+
+      <ScrollView contentContainerStyle={[styles.container, { paddingBottom: insets.bottom }]}>
+        <Text style={styles.title}>{t('jumpstartIntro.title')}</Text>
+
+        <View style={styles.description}>
+          <Text style={styles.descriptionLine}>{t('jumpstartIntro.infoLine1')}</Text>
+          {noTokens && <Text style={styles.descriptionLine}>{t('jumpstartIntro.infoLine2')}</Text>}
+        </View>
+
+        <Button
+          onPress={onButtonClick}
+          text={t(noTokens ? 'jumpstartIntro.addFundsCelo.cta' : 'jumpstartIntro.haveFundsButton')}
```

Comment:
we have some hopes and dreams to eventually do static analysis on translation keys, so for this reason we try to avoid dynamically calculated translation keys inside `t`. so it'd be great to move the ternary outside of the `t` function